src/components/sbb-divider/sbb-divider.stories.js

```diff
@@ -0,0 +1,101 @@
+import { h } from 'jsx-dom';
+import readme from './readme.md';
+import {
+  SbbColorBlackDefault,
+  SbbColorWhiteDefault,
+} from '@sbb-esta/lyne-design-tokens/dist/js/sbb-tokens';
+
+const documentationPlatformContainerStyle = (context) => {
+  if (context.args.negative) {
+    return {};
+  }
+
+  return `background-color: ${SbbColorWhiteDefault};`;
+};
+
+const wrapperStyle = (context) => {
+  if (context.args.negative) {
+    return `background-color: ${SbbColorBlackDefault};`;
+  }
+
+  return `background-color: ${SbbColorWhiteDefault};`;
+};
+
+const Template = (args) => (
+  <div style="height: 340px; padding: 20px;">
+    <sbb-divider {...args} />
+  </div>
+);
+
+const orientation = {
+  control: {
+    type: 'select',
+  },
+  options: ['horizontal', 'vertical'],
+};
+
+const negative = {
+  control: {
+    type: 'boolean',
+  },
+  table: {
+    category: 'Appearance',
+  },
+};
+
+const defaultArgTypes = {
+  orientation,
+  negative,
+};
+
+const defaultArgs = {
+  orientation: orientation.options[0],
+  negative: false,
+};
+
+export const divider = Template.bind({});
+
+divider.args = JSON.parse(JSON.stringify(defaultArgs));
```

Comment:
Can you maybe add a story for the vertical orientation? When I was checking the chromatic report I saw that this doesn't exist, but for visual regression tests I think it's better to have this variant too. Thank you!